packages/webapp/components/footer/FooterNavBarTabs.tsx

```diff
@@ -82,12 +88,45 @@ const Tab = ({ tab, isActive }: TabProps) => {
   );
 };
 
+const Notifications = ({ active }: { active: boolean }): JSX.Element => {
+  const { unreadCount } = useNotificationContext();
+
+  return (
+    <div className="relative">
+      <BellIcon secondary={active} size={IconSize.Medium} />
+      {!!unreadCount && (
+        <Bubble
+          className={classNames('-right-1.5 -top-1.5 cursor-pointer px-1')}
+        >
+          {getUnreadText(unreadCount)}
+        </Bubble>
+      )}
+    </div>
+  );
+};
```

Comment:
I wanted to re-use the `NotificationsBell` component, but it adds a tooltip and an extra button that doesn't fit in the bottom navbar. So for experiment sake, I thought of just doing a local component here.